app/controllers/account_controller.rb

```diff
@@ -111,17 +103,38 @@ def strip_new_user_param(arg)
   # where "evil" means: sending a Verification email will throw an error;
   # the Verification email will cause Undelivered Mail Returned to Send; and/or
   # it's a known spammer.
-  def block_evil_signups!
-    return false unless evil_signup_credentials?
-
-    true
+  def abort_signup_with_client_error
+    # Too Many Requests == 429. Any 4xx status (Client Error) would also work.
+    render(status: :too_many_requests,
+           content_type: "text/plain",
+           plain: "We grow weary of this. Please go away.")
```

Comment:
I seem to remember one innocent user actually receiving this message accidentally.  Who knows how many were so offended by it that we never heard from them!  Maybe it's time to soften it?  "We've received a lot of malicious sign-up attempts that match your chosen login name or email address.  Please contact us if you are not a robot."  Would that work okay?

Reply:
Good point. I'll change the message.